### ISSUE TITLE
PyRDP running error on Windows 10

### ISSUE THREAD

**Issue by yraiesets** [GoSecure/pyrdp#316]
Both pyrdp-mitm.py and pyrdp-player.py trigger a TypeError on execution.

Platform : Windows 10.
Python Version : 3.9.5

`Traceback (most recent call last):
  File "C:\Users\admin\Desktop\pyrdp\bin\pyrdp-mitm.py", line 15, in <module>
    asyncioreactor.install(asyncio.get_event_loop())
  File "C:\Users\admin\AppData\Local\Programs\Python\Python39\lib\site-packages\twisted\internet\asyncioreactor.py", line 307, in install
    reactor = AsyncioSelectorReactor(eventloop)
  File "C:\Users\admin\AppData\Local\Programs\Python\Python39\lib\site-packages\twisted\internet\asyncioreactor.py", line 60, in __init__
    raise TypeError(
TypeError: SelectorEventLoop required, instead got: <ProactorEventLoop running=False closed=False debug=False>`

**Comment by obilodeau** [comment]
Something related. I think most of the integration tests on Windows are worthless. Even if the return code is 0 check the individual outputs.

**Comment by obilodeau** [comment]
Tonight on Windows 10 with Python 3.10.10:

```
python venv\Scripts\pyrdp-player.py
Traceback (most recent call last):
  File "C:\[...]\tools\pyrdp\venv\Scripts\pyrdp-player.py", line 7, in <module>
    exec(compile(f.read(), __file__, 'exec'))
  File "C:\[...]\tools\pyrdp\bin\pyrdp-player.py", line 11, in <module>
    asyncioreactor.install(asyncio.new_event_loop())
  File "C:\[...]\tools\pyrdp\venv\lib\site-packages\twisted\internet\asyncioreactor.py", line 306, in install
    reactor = AsyncioSelectorReactor(eventloop)
  File "C:\[...]\tools\pyrdp\venv\lib\site-packages\twisted\internet\asyncioreactor.py", line 61, in __init__
    raise TypeError(
TypeError: ProactorEventLoop is not supported, got: <ProactorEventLoop running=False closed=False debug=False>
```

**Comment by obilodeau** [comment]
The PyRDP Player works if you add this line below the `import asyncio` call at the top of the `bin/pyrdp-player.py` file:

```python
asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
```

Not sure what the real fix will be at this point but an OS-specific reactor could do it.

**Comment by obilodeau** [comment]
The fix for the Twisted problem with Python 3.8+ on Windows seems to have been merged in 2020: https://github.com/twisted/twisted/pull/1338.

Testing it

**Comment by obilodeau** [comment]
This [cc82cc8](https://github.com/GoSecure/pyrdp/pull/439/commits/cc82cc8eb6c06c549db118b4715ca37f622fe49a) is attempting to package a fix that will not cause problems elsewhere. Doing only the player for now but I might need to revisit this assumption later.

**Comment by obilodeau** [comment]
Fixed by #439
